static/docs/commands-reference/pull.md

```diff
@@ -1,53 +1,123 @@
 # pull
 
-Pulls data files to the working space.
+Pulls data files to the local workspace from a remote workspace.
 
-The set of data files to pull (usually it means downloading from the remote
-storage if file is not in the local cache yet) is determined by analyzing all
-`.dvc` files in the current branch, unless `--all-branches` is specified.
+## Usage
 
-The command pulls only output of a specific stage if dvc file is specified `dvc pull data.zip.dvc`.
+```usage
+    usage: dvc pull [-h] [-q | -v] [-j JOBS]
+                    [--show-checksums] [-r REMOTE]
+                    [-a] [-T] [-d] [-f] [-R]
+                    [targets [targets ...]]
+
+    Pull data files from the cloud.
+
+    positional arguments:
+      targets               DVC files.
+```
+
+## Description
+
+DVC supports sharing a data cache between workspaces.  Members of a team may
```

Comment:
Double spaces :)

Reply:
Yes.  I always put two spaces after the end of a sentence.  That's the norm, or at least that was the norm when I learned how to type.

Reply:
Ha. Gotcha! :) I was not even aware about this, and was thinking this is a bug. I would still vote for one space style - just because it's hard to maintain two spaces and it would require changing a lot of other docs. I think one space is considered as a norm as well now.